viz/cmd/allow-scrapes.go

```diff
@@ -0,0 +1,143 @@
+package cmd
+
+import (
+	"os"
+	"text/template"
+
+	pkgcmd "github.com/linkerd/linkerd2/pkg/cmd"
+	"github.com/linkerd/linkerd2/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+const (
+	allowScrapePolicy = `---
+apiVersion: policy.linkerd.io/v1beta1
+kind: Server
+metadata:
+  name: proxy-admin{{ if .TargetNs }}\n  namespace: {{ .TargetNs }}{{end}}
```

Comment:
Looks like the newline isn't getting escaped as desired here and is rendering as a literal backslash n:

```
bin/linkerd viz allow-scrapes -n emojivoto | grep namespace
  name: proxy-admin\n  namespace: emojivoto
  name: proxy-metrics\n  namespace: emojivoto
  name: proxy-probes\n  namespace: emojivoto
  name: prometheus-scrape\n  namespace: emojivoto
      namespace: linkerd-viz
  name: proxy-probes\n  namespace: emojivoto
      namespace: linkerd-viz
```